src/templates/platform.js

```diff
@@ -32,17 +42,19 @@ function PlatformTemplate({ data }) {
             pad={{ vertical: 'large', horizontal: 'large' }}
             direction="column"
           >
-            <Heading
-              margin={{
-                bottom: 'medium',
-                left: 'none',
-                right: 'none',
-                top: 'none',
-              }}
-            >
-              {title}
-            </Heading>
-            {aside && <MarkdownLayout>{aside}</MarkdownLayout>}
+            <Link to={homeRoute}>
+              <Heading
+                margin={{
+                  bottom: 'medium',
+                  left: 'none',
+                  right: 'none',
+                  top: 'none',
+                }}
+              >
+                {title}
+              </Heading>
+            </Link>
+            {aside && <Aside>{aside}</Aside>}

```

Comment:
The long links in the simplivity aside are forcing this parent box to be way to wide causing the main doc to move below this aside in too many cases. Maybe the <Aside> needs to be tweaked to limit its width sooner and wrap the links

Reply:
can do